docs/design/diagram-types/README.md

```diff
@@ -0,0 +1,35 @@
+# Diagram Types
+
+Creating and maintaining diagrams is a challenge for any team. Common reasons across these challenges include:
+
+- Not leveraging tools to assist in generating diagrams
+- Uncertainty on what to include in a diagram and when to create one
+
+Overcoming these challenges and effectively using design diagrams can amplify a team's ability to execute throughout the entire Software Development Lifecycle, from the design phase when proposing various designs to leveraging it as documentation as part of the maintenance phase.
+
+This section will share sample tools for diagram generation, provide a high level overview of the different types of diagrams and provide examples of some of these types.
+
+There are two primary classes of diagrams:
+
+1. Structural
```

Comment:
Is this meant to be an ordered list? if not, recommend moving to an unordered list